stdlib/src/zlib.rs

```diff
@@ -53,6 +54,26 @@ mod zlib {
     #[pyattr]
     const DEF_MEM_LEVEL: u8 = 8;
 
+    #[cfg(feature = "zlib")]
+    #[pyattr(name = "ZLIB_RUNTIME_VERSION", once)]
+    fn zlib_runtime_version(_vm: &VirtualMachine) -> String {
+        unsafe {
+            CStr::from_ptr(libz_sys::zlibVersion())
+                .to_string_lossy()
+                .into_owned()
+        }
+    }
+
+    #[cfg(feature = "zlib")]
+    #[pyattr(name = "ZLIB_VERSION", once)]
+    fn zlib_version(_vm: &VirtualMachine) -> String {
+        unsafe {
+            CStr::from_ptr(libz_sys::zlibVersion())
+                .to_string_lossy()
+                .into_owned()
+        }
+    }
+
```

Comment:
```suggestion
```

By looking [zlib.h](https://github.com/madler/zlib/blob/master/zlib.h), `ZLIB_VERSION` is the version from header and `zlibVersion` is the version from library.